src/activation/useActivateRecipe.ts

```diff
@@ -102,25 +104,29 @@ function useActivateRecipe(
           recipe,
           serviceAuths
         );
-        const isPermissionsAcceptedByUser =
-          await ensurePermissionsFromUserGesture(recipePermissions);
-        if (!isPermissionsAcceptedByUser) {
-          if (source === "extensionConsole") {
-            // Note: The prefix "Marketplace" on the telemetry event name
-            // here is legacy terminology from before the public marketplace
-            // was created. It refers to the mod-list part of the extension
-            // console, to distinguish that from the workshop.
-            // It's being kept like this so our metrics history stays clean.
-            reportEvent(Events.MARKETPLACE_REJECT_PERMISSIONS, {
-              ...selectActivateEventData(recipe),
-              reactivate: isReactivate,
-            });
-          }
 
-          return {
-            success: false,
-            error: "You must accept browser permissions to activate.",
-          };
+        if (checkPermissions) {
+          const isPermissionsAcceptedByUser =
```

Comment:
I'm not confident about this suggestion, but the original var name isn't grammatically correct 🤷‍♀️
```suggestion
          const hasUserAcceptedPermissions =
```